pkg/controller/controller_broker_test.go

```diff
@@ -156,6 +156,9 @@ func TestShouldReconcileBroker(t *testing.T) {
 }
 
 func TestReconcileBrokerExistingServiceClass(t *testing.T) {
+	// Verify the fake clients perform as expected during reconcileBroker().
+	// There must be exactly 1 Broker action, 2 Catalog actions
+	// and no kube resources created
 	fakeKubeClient, fakeCatalogClient, fakeBrokerClient, testController, sharedInformers := newTestController(t, getTestCatalogConfig())
 
 	testServiceClass := getTestServiceClass()
```

Comment:
I think this line below forces the informer to spit out an update for the reconciler to see something has happened and thus update the catalog. (I am unclear how the informer would know about this without the update already occurring. perhaps I am getting my causes and effects mixed up).

thus we poll the broker for it's catalog, to do that the catalogclient (the thing that talks to the svc-cat apiserver, which is the controller calling that client) will do an update of the serviceclass and update the broker as a whole.

there should be no core resources touched because the core is not involved here.